src/render.jl

```diff
@@ -36,7 +36,8 @@ While the return value is a `struct`, it is callable and can be used as the
 """
 function FlameColors(n::Integer=2;
                      colorbg=colorant"white", colorfont=colorant"black",
-                     colorsrt=colorant"crimson", colorsgc=colorant"orange")
+                     colorsrt=colorant"crimson", colorsgc=colorant"orange",
+                     darkmode=false)
```

Comment:
How about putting `darkmode` as the first keyword and then making `colorbg` and `colorfont` dependent upon it?
```julia
julia> f(; darkmode=false, bg=(darkmode ? "black" : "white")) = bg
f (generic function with 1 method)

julia> f()
"white"

julia> f(; darkmode=true)
"black"
```
(You don't actually need the parens around the ternary, I just find it slightly clearer.)

Reply:
sounds good. done